src/Logger/Adapter.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Utopia\Logger;
+
+use Exception;
+
+abstract class Adapter
+{
+    /**
+     * Get unique name of an adapter
+     *
+     * @return string
+     */
+    abstract public static function getAdapterName(): string;
```

Comment:
```suggestion
    abstract public static function getName(): string;
```

The scope already suggests this is the adapter name.

Reply:
Renamed all to `getName`.